src/users/components/ProfileUI.vue

```diff
@@ -62,6 +62,14 @@
           icon="fas fa-comments"
           @click="$emit('detail', user)"
         />
+        <QBtn
+          v-if="triggerCriteria"
```

Comment:
rename to `canReport`?

I think it would be nice to keep the button but explain _why_ the user can't be reported. maybe via tooltip (bad for mobile) or dialog.